app/views/groupings/show.html.erb

```diff
@@ -0,0 +1,35 @@
+<%= render 'organizations/organization_banner' %>
+
+<div class="site-content">
+  <div class="site-content-cap">
+    <h2 class="site-content-heading"><%= @grouping.title %></h2>
+  </div>
+
+  <div class="site-content-body">
+    <% if @grouping.groups.present? %>
+      <div class="team-grid">
+        <div class="columns">
+          <div class="one-half column">
+            <% @grouping.groups.each do |group| %>
+                <% if cycle('odd', 'even') == 'odd' %>
+                    <%= render partial: "group", locals: { group: group } %>
+                <% end %>
+            <% end %>
+          </div>
+          <div class="one-half column">
+            <% @grouping.groups.each do |group| %>
+                <% if cycle('odd', 'even') == 'even' %>
+                    <%= render partial: "group", locals: { group: group } %>
+                <% end %>
+            <% end %>
+          </div>
```

Comment:
Hi @jasonlong, we're implementing a team management interface here. Teachers will be able to move students from a team to another by simply drag and drop the student avatars. 

It would be great if you can give me some feedback on this UI. Thanks!

/cc @johndbritton @tarebyte